esp/eclwatch/ws_XSLT/nls/en/hpcc.xml

```diff
@@ -156,6 +157,7 @@
 <st id='ViewingPage'>Viewing page</st>
 <st id='ViewLogFile'>View Log File</st>
 <st id='ViewAuditLogFile'>View Audit Log File</st>
+<st id='ViewSparkClusterInfo'>View Spark Cluster Information</st>
```

Comment:
Depending on the context in which this message is used, it might need to to change to:
"View Spark-Thor Cluster Information"
or
"View Integrated Spark Cluster Information"

Reply:
Ok, I will change it to "View Spark-Thor Cluster Information".